api/api.yaml

```diff
@@ -292,6 +292,31 @@ paths:
           $ref: '#/components/responses/400'
         '500':
           $ref: '#/components/responses/500'
+  /v1/{identifier}/claims/{id}/qrcode:
+    get:
+      summary: Get Claim QR code
+      operationId: GetQrCodeClaim
```

Comment:
I would put `GetClaimQrCode` instead of `GetQrCodeClaim`

Reply:
The same with the object response (`GetQrCodeClaimResponse`)

Reply:
Done